.github/ISSUE_TEMPLATE/meeting-sprint-planning.md

```diff
@@ -24,28 +20,18 @@ Our goal is to synchronize the team on the most important things to work on, and
 
 _Follow the agenda below, checking boxes as we complete each step._
 
-## Before the meeting
-
-- [ ] Create this week's sprint board
-- [ ] Choose a sprint meeting facilitator
-
-## Review of last week's Sprint Board. (5m)
-
-- [ ] Team members have all filled out last week's team sync
-- [ ] Decide what to do with any incomplete deliverables
-- [ ] Archive last week's board
-
-## Populate this week's Sprint Board (45m)
-
-- [ ] Archive and celebrate closed deliverables 🎉
-- [ ] Team members note any reduced availability
-- [ ] Add new deliverables to Sprint Board
-- [ ] Somebody is assigned to each deliverable
-- [ ] Everybody agrees the current cycle's plan is realistic and that the right items are on the board.
-
-## Support ticket overview. (10m)
 
-- [ ] Support steward asks for assistance on issues that need it.
-- [ ] (_once a month_) Transition support steward roles to the next team member
-  - [ ] Open issue for next Support Steward.
-  - [ ] Close previous issue for Support Steward.
+- [ ] Review of last week's Sprint Board. (10m)
```

Comment:
There are some "week" references in this section that should be modified with the biweekly cadence.